js/src/forum/ForumApplication.ts

```diff
@@ -73,10 +73,14 @@ export default class ForumApplication extends Application {
    */
   discussions: DiscussionListState = new DiscussionListState({});
 
+  route: typeof Application.prototype.route & ReturnType<typeof makeRouteHelpers>;
+
   constructor() {
     super();
 
     routes(this);
+
+    this.route = Object.assign(Application.prototype.route.bind(this), makeRouteHelpers(this));
```

Comment:
We may want to use `this.constructor.route` instead? That way we don't make it specific to Application in case for some reason some extension wants to modify it. Even if not for that, just to make sure we're referencing the current subclass just in case.

Reply:
That exact syntax didn't work, but I changed it to use prototypes